src/main/java/flaxbeard/steamcraft/block/BlockBoiler.java

```diff
@@ -298,13 +304,14 @@ public ItemStack getPickBlock(MovingObjectPosition target, World world, int x, i
     }
 
     @Override
-    public boolean onWrench(ItemStack stack, EntityPlayer player, World world,
-                            int x, int y, int z, int side, float xO, float yO, float zO) {
+    public boolean onWrench(ItemStack stack, EntityPlayer player, World world, int x, int y, int z, int side, float xO,
```

Comment:
I don't think either the old nor the new styles are any better than one another; they both break the codestyle. Granted, yours is somewhat closer to our style, it should still all be on a single line.